sign/schnorr/schnorr_test.go

```diff
@@ -125,3 +126,19 @@ func TestSchnorrMalleability(t *testing.T) {
 	err = Verify(suite, kp.Public, msg, s)
 	assert.Error(t, err, "schnorr signature malleable")
 }
+
+func FuzzSchnorr(f *testing.F) {
+	f.Fuzz(func(t *testing.T, msg []byte) {
+		if (len(msg) < 1) || (len(msg) > 1000) {
+			t.Skip("msg must have byte length between 1 and 1000")
+		}
+		suite := edwards25519.NewBlakeSHA256Ed25519()
+		kp := key.NewKeyPair(suite)
```

Comment:
this should probably be outside of the fuzzing method since it doesn't need to be re-instantiated for each fuzzing pass